### ISSUE TITLE
Providing emission_profile_upstream_product file

### ISSUE THREAD

**Issue by AnneSchoenauer** [2DegreesInvesting/tiltWorkflows#113]
Dear Sven, 
Could you please provide the emission_profile_upstream_product file in the output table folder that you have created still? I created a ticket for Tilman and me to not forget to try out DataBricks on our own but for now it would be great if you could still create it. 
Thanks a lot!
Best
Anne

**Comment by SKruthoff** [comment]
Hi, 

I will upload the file just be aware that it is around 60GB.

**Comment by Tilmon** [comment]
Ufff... In that case, maybe a subset is enough for review. What do you think @AnneSchoenauer ?

**Comment by AnneSchoenauer** [comment]
Yes maybe right! Maybe you could pick like 100 random companies? the rest is covered by the data quality checks and captured in the dashboard! Thanks @SKruthoff

**Comment by SKruthoff** [comment]
I can also do that. But I see that we have a meeting next week around using databricks. You could also use that to do some analysis on the full dataset, since the data available in databricks would be that data I am sharing.

**Comment by Tilmon** [comment]
@AnneSchoenauer this ticket we can close right?
